index.js

```diff
@@ -166,15 +165,22 @@ module.exports = function (content) {
 
   var cb = this.async();
 
-  var publicPath = options.publicPath || (webpackOptions.output && webpackOptions.output.publicPath) || '/';
+  const publicPath = typeof options.publicPath === 'string'
```

Comment:
Can you change this whole thing to `if...else`, it's really hard to understand what you're doing here

Reply:
changed to `if...else`